pkg/util/test/util.go

```diff
@@ -888,6 +856,135 @@ func validateComponents(pxImageList map[string]string, cluster *corev1.StorageCl
 	return nil
 }
 
+func validateStork(pxImageList map[string]string, cluster *corev1.StorageCluster, k8sVersion string, timeout, interval time.Duration) error {
```

Comment:
do we need to validate the configmap / storageclass / service and other components that created by operator as well? probably just the existence for now if no particular specs validations are needed for those?

Reply:
I don't know this is basic validation, I honestly don't even check that when in my manual regression. We have many jobs that are running with stork, if there was something wrong there we would have known. Lets move those out to a second, not P1 validations? What do you think?

Reply:
sounds good